models/School.js

```diff
@@ -5,12 +5,15 @@ const SchoolSchema = new mongoose.Schema({
   name: {
     type: String,
   },
-  Address: {
+  address: {
     type: String,
   },
   phoneNumber: {
     type: Number,
   },
+  riderId: {
+    type: Array,
```

Comment:
type : ObjectId

Reply:
it's an array. We need to have all rides for a school :)